lib/rules/subsystem.js

```diff
@@ -6,6 +6,7 @@ const validSubsystems = [
   'benchmark'
 , 'build'
 , 'bootstrap'
+, 'cli',
```

Comment:
This list is using leading commas, so the trailing comma needs to be removed.

```suggestion
, 'cli'
```